src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -370,7 +370,6 @@ public static void AddSchemaFilterInstance<TFilter>(
             if (filterInstance == null) throw new ArgumentNullException(nameof(filterInstance));
             swaggerGenOptions.SchemaFilterDescriptors.Add(new FilterDescriptor
             {
-                Type = typeof(TFilter),
```

Comment:
There is not need to remove this assignments to property `Type`, even when this pass all the tests. Maybe someone out there is relying on the `Type` being assigned even when non needed by the library, because it is only used if the `FilterInstance` is not assigned.

I would revert the changes in this file.

Reply:
Please revert these changes - as you say, they are part of the visible external behaviour and someone might be using them.